### PR TITLE
LPS-141750

### PR DESCRIPTION
**Pull request by catarineaguiar**
https://issues.liferay.com/browse/LPS-143173

### CODE REVIEW

**Comment by catarineaguiar** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/macros/Masterclass.macro [hunk 1]
```diff
@@ -25,6 +25,54 @@ definition {
 			siteTitle = "${siteName}");
 	}
 
+	macro createAccount {
```
Comment (david-gutierrez-mesa) [inline modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/macros/Masterclass.macro:28]: don't we have a Jason method for this?

File: modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/tests/Masterclass.testcase [hunk 2]
```diff
@@ -582,6 +582,76 @@ definition {
...
+		}
+
+		task ("Create a New Account") {
+			Masterclass.createAccount(
+				emailAddress = "user@liferay.com",
+				firstName = "User",
+				lastName = "User",
+				password = "test",
+				screenName = "User");
+		}
```
Comment (david-gutierrez-mesa) [inline modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/tests/Masterclass.testcase:609]: We should do an assert here in this test. to ensure the account was created.

**Review by david-gutierrez-mesa — CHANGES_REQUESTED** [review]
Nice job!

File: modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/macros/Masterclass.macro (hunk 1 above)
Comment (catarineaguiar) [reply modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/macros/Masterclass.macro:28]: But the testcase is to create the account by UI, right?

**Comment by catarineaguiar** [comment]
ci:test:sf

**Comment by david-gutierrez-mesa** [comment]
Just started reviewing :)

**Comment by david-gutierrez-mesa** [comment]
Rebased. See #231

**Pull request closed without merging**